torch/csrc/jit/script/compiler.cpp

```diff
@@ -2636,14 +2636,24 @@ struct to_ir {
       args.emplace_back(loc, "end", emitExpr(Expr(slice.end().get())));
     }
     if (input->type()->cast<TupleType>()) {
+
+      auto has_step = slice.step().present();
+      // TODO: add support for slicing tuples
+      if (has_step)
+      {
+        throw ErrorReport(loc) << "Unsupported operation: slicing tuples isn't supported";
+      }
+
       if (has_end) {
         return emitTupleSlice(loc, args[0], args[1], /*end*/ args[2]);
       } else {
         return emitTupleSlice(loc, args[0], args[1], c10::nullopt);
       }
     }
+
+    auto stride = emitExpr(Expr(slice.stepOr(1)));
```

Comment:
nit: "stride" is a heavyweight term in pytorch, let's avoid it and stick with step